share-coordinator/src/main/java/org/apache/kafka/coordinator/share/ShareCoordinatorShard.java

```diff
@@ -814,7 +814,7 @@ private Optional<ReadShareGroupStateResponseData> maybeGetReadStateError(ReadSha
 
         SharePartitionKey mapKey = SharePartitionKey.getInstance(groupId, topicId, partitionId);
         if (leaderEpochMap.containsKey(mapKey) && leaderEpochMap.get(mapKey) > partitionData.leaderEpoch()) {
-            log.error("Request leader epoch id is smaller than last recorded.");
+            log.error("Request leader epoch id is smaller than last recorded current: {}, requested:{}.", leaderEpochMap.get(mapKey), partitionData.leaderEpoch());
```

Comment:
For consistency, `Request leader epoch smaller than`....

Reply:
will also add prefix mentioning type of request